docs/_documentations/installoncloud.md

```diff
@@ -40,7 +40,7 @@ Complete the following steps after you install `chectl`:
 
 1. Download the [codewind-che checluster yaml](https://github.com/eclipse/codewind-che-plugin/blob/master/setup/install_che/che-operator/codewind-checluster.yaml) file to your machine.
     - You can modify this file, but leave the `spec.server.cheWorkspaceClusterRole` field set to `eclipse-codewind` and the `spec.storage.preCreateSubPaths` field set to `true`.
-2. If you install on Kubernetes, determine your Ingress domain. If you're unsure of your Ingress domain, ask your cluster administrator.
+2. If you're installing on a Kubernetes platform other than OpenShift, determine your Ingress domain. If you're unsure of your Ingress domain, ask your cluster administrator.
     - Set the `spec.server.ingressDomain` field in the Che .yaml to the Ingress domain.
 2. Install Che:
```

Comment:
Do we have two step 2's?
I just checked on the website (https://www.eclipse.org/codewind/installoncloud.html), and the numbering looks good there, though.

Reply:
@sishida Thanks for catching that, I fixed it. Looks like Github automatically fixes duplicate numbering when displaying Markdown.